advanced_topics.qmd

```diff
@@ -3,13 +3,16 @@
 {{< include _knitr_hooks.qmd >}}
 
 ```{r}
+#| label: setup-advanced-data-manip
 #| include: FALSE
 library(arrow)
 library(dplyr)
-```
-```{r}
-#| label: setup-advanced-data-manip
-#| include: false
+library(vctrs)
+library(ggplot2)
+library(see)
```

Comment:
What is this used for?

Reply:
This is for the half violin. We could also copy pasta from stack overflow (and there's at least one other implementation), but I didn't want to (re)invent anything here

Reply:
Ah ok cool. Maybe we can see:: reference the function we use there, just so it's clear in the code example that it's not a vanilla ggplot2 function?